### ISSUE TITLE
Do amazingly good people but non-Christians go to hell?

### ISSUE THREAD

**Issue by martyychang** [martyychang/christianity-in-24#20]
If someone who was a good person by most standards dies, and that person had never heard the Gospel, is that person doomed to hell?

**Comment by ahrobinson** [comment]
This is a tough question mainly because, IMO, it forces you to really ask yourself if you believe in the goodness, and righteousness of God. If someone who has never heard the Gospel dies and goes to hell, would we still say that God is good and also justified in doing so? Romans 1 seems to state that no man has an excuse regardless if they have heard the Gospel or not because God makes Himself known in all of creation.

**Comment by DavidLozzi** [comment]
I would agree. Who defines "amazingly good"? Where's the standard? My idea of good differs from yours, guarantee it, since it's subjective. God has set the line, the standard, and we can't make it without him. 

'For ever since the world was created, people have seen the earth and sky. Through everything God made, they can clearly see his invisible qualities—his eternal power and divine nature. So they have no excuse for not knowing God. They know the truth about God because he has made it obvious to them. '

Romans 1:19-20
https://www.bible.com/bible/116/ROM.1.19-20

**Comment by ghost** [comment]
Good and evil are qualifiers, not quantifiers - what are the metrics? 

We’re also forced to challenge the idea of hell as being a construct of the Catholic Church very specifically, as the Hebrew Scriptures “second death” is your soul not moving forward into eternal living - no torment or pain, just flatline and gone. I would argue that’s what Paul meant as well when writing about it, not forever punishment, just your existence finally vanishing. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by corachang86** [comment]
Addressed in Hour 20 (Hard Questions) — God shows no partiality (Acts 10:34-35), judgment is by action not belief — and Hour 23 (Death and What Comes After) — universal restoration as our stated belief. No one is irredeemable.
